README.rst

```diff
@@ -39,36 +39,38 @@ PyAEDT
 
 What is PyAEDT?
 ---------------
-PyAEDT is a Python library that interacts directly with the AEDT API
-to make scripting simpler for the end user. Its architecture
-can be reused for all AEDT 3D products (HFSS, Icepak, Maxwell 3D, and
-Q3D Extractor), 2D tools, and Ansys Mechanical. It also provides support for circuit
-tools like Nexxim and system simulation tools like Twin Builder. Finally it provides
-scripting capabilities in Ansys layout tools like HFSS 3D Layout and EDB. Its class
-and method structures simplify operation for the end user while reusing information
-as much as possible across the API.
+PyAEDT is a Python library that interacts directly with the API for
+Ansys Electronics Desktop (AEDT) to make scripting simpler. The architecture
+for PyAEDT can be reused for all AEDT 3D products (HFSS, Icepak, Maxwell 3D,
+and Q3D Extractor), 2D tools, and Ansys Mechanical. PyAEDT also provides
+support for circuit tools like Nexxim and system simulation tools like
+Twin Builder. Finally, PyAEDT provides scripting capabilities in Ansys layout
+tools like HFSS 3D Layout and EDB. The PyAEDT class and method structures
+simplify operation while reusing information as much as possible across
+the API.
 
 Install on CPython from PyPI
 ----------------------------
-You can install PyAEDT on CPython 3.7 through 3.10 from PyPI:
+You can install PyAEDT on CPython 3.7 through 3.10 from PyPI with this command:
 
 .. code:: python
 
     pip install pyaedt
 
-Install PyAEDT with all extra packages (matplotlib, numpy, pandas, pyvista):
+To install PyAEDT with all extra packages (Matplotlib, NumPy, Pandas, and PyVista),
+use this command:
 
 .. code:: python
 
     pip install pyaedt[full]
```

Comment:
or pip install pyaedt[all]

Reply:
@cutelolly I had already merged this branch, but I think knowing this alternatively command is useful. Feel free to add it in a PR of your own!